libdebug/state/debugging_context.py

```diff
@@ -233,7 +247,26 @@ def dead(self) -> bool:
         """
 
         return not self._threads
+    
+    @property
```

Comment:
Do we need a property for this if we don't check what the user is setting the value to?